src/components/MakePayment.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from "react";
+import Button from "./Button";
+import "./CalcPayment.css";
+
+const MakePayment = ({ rates, paymentData, setPaymentData }) => {
+  const [selectedCurrency, setSelectedCurrency] = useState("GBP");
+  const [amount, setAmount] = useState("0.00");
+  const [description, setDescription] = useState("");
+
+  const selectCurrency = (event) => {
+    const currency = event.target.value;
+    setSelectedCurrency(currency);
+  };
+
+  const amountInput = (event) => {
+    setAmount(event.target.value);
+  };
+
+  const descriptionInput = (event) => {
```

Comment:
Same as above. Names are hard to pick, but very important for a fellow developer that may land in the codebase and will try to make sense of the code ;)